README.md

```diff
@@ -122,6 +122,9 @@ DataYoga supports both SQL and [JMESPath](https://jmespath.org/) expressions. JM
 
 DataYoga adds the following custom functions to the standard JMESPath library:
 
-| Function | Description                                     | Example                                                                                                         | Comments                                                                               |
-| -------- | ----------------------------------------------- | --------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------- |
-| `concat` | Concatenates an array of variables or literals. | Input: `{fname: "john", lname: "doe"}` <br /> Expression: `concat([fname,' ',lname])` <br /> Output: `john doe` | This is equivalent to the more verbose built-in expression: `' '.join([fname,lname])`. |
+| Function     | Description                                                             | Example                                                                                                         | Comments                                                                               |
+| ------------ | ----------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------- |
+| `capitalize` | Capitalizes all the words in the string                                 | Input: `{name: "john doe"}` <br /> Expression: `capitalize(name)` <br /> Output: `John Doe`                     |
+| `concat`     | Concatenates an array of variables or literals                          | Input: `{fname: "john", lname: "doe"}` <br /> Expression: `concat([fname,' ',lname])` <br /> Output: `john doe` | This is equivalent to the more verbose built-in expression: `' '.join([fname,lname])`. |
+| `lower`      | Converts all uppercase characters in a string into lowercase characters | Input: `{fname: "John"}` <br /> Expression: `upper(fname)` <br /> Output: `john`                                |
```

Comment:
example should be 'lower' (instead of 'upper')

Reply:
thanks, my bad.